manifests/t/Tutanota/Tutanota/3.114.1/Tutanota.Tutanota.installer.yaml

```diff
@@ -0,0 +1,30 @@
+# Created with Komac v1.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Tutanota.Tutanota
+PackageVersion: 3.114.1
+Platform:
+- Windows.Desktop
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2023-06-14
+AppsAndFeaturesEntries:
+- DisplayName: Tutanota Desktop 3.114.1

```

Comment:
```suggestion
```
We can probably get rid of these since they `Publisher` and `PackageName` give a good match already. These would need to be manually updated for each new update